node/node.go

```diff
@@ -374,19 +383,50 @@ func (n *Node) initNetworking(primaryNetVdrs validators.Set) error {
 	return err
 }
 
+type NodeRuntimeState struct {
+	// The process id of the node
+	PID int
+	// URI to access the node API
+	// Format: [https|http]://[host]:[port]
+	URI string
+	// Address other nodes can use for bootstrapping
+	// Format: [host]:[port]
+	BootstrapAddress string
```

Comment:
Not really a huge fan of the naming here - Should this be `P2PAddress`, `StakingAddress` or something else? `BootstrapAddress` feels very specific to what inspired this PR (rather than what this PR actually _is_) i.e. this isn't an address specific to bootstrapping.